solvebio/cli/data.py

```diff
@@ -688,7 +688,8 @@ def _download_recursive(
             "{}Downloading file {}".format("[Dry run] " if dry_run else "", local_path)
         )
         if not dry_run:
-            os.makedirs(parent_dir, exist_ok=True)
```

Comment:
os.makedirs doesn't have the parameter `exist_ok` in python 2.7, so it would fail. This only works in python >3